src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,144 @@
+/**
+* @title WET-BOEW Exit script plugin
+* @overview Plugin redirects users to non secure site
+* @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+* @author @ipaksc
+*/
+( function( $, window, wb ) {
+"use strict";
+var componentName = "wb-exitscript",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	defaults = {},
+	i18nDict = {
+		en: {
+			"exitscript-warning": "Warning",
+			"exitscript-exitMsg": "You are about to leave a secure site, do you wish to continue?",
+			"exitscript-yesMsg": "Yes",
+			"exitscript-cancelMsg": "Cancel"
+		},
+		fr: {
+			"exitscript-warning": "Avertissement",
+			"exitscript-exitMsg": "Vous êtes sur le point de quitter un site sécurisé. Voulez-vous continuer?",
+			"exitscript-yesMsg": "Oui",
+			"exitscript-cancelMsg": "Annuler"
+		}
+	},
+
+/**
+* @method init
+* @param {jQuery Event} event Event that triggered the function call
+*/
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			$elm,
+			settings;
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+		true,
+		defaults, {},
+		window[ componentName ],
+		wb.getData( $elm, componentName )
+	);
+			i18nDict = i18nDict[ $( "html" ).attr( "lang" ) || "en" ];
```

Comment:
You should be using the following instead of scrapping the DOM
```suggestion
			i18nDict = i18nDict[ wb.lang || "en" ];
```

Reply:
It's fixed